File: test_scripts/r/test1/independent.R
# Independent script with no dependencies

# Simple standalone script
message("Starting independent analysis")

# Generate some data
x <- 1:10
y <- x^2 + rnorm(10)

# Simple analysis
correlation <- cor(x, y)
print(paste("Correlation:", round(correlation, 3)))

# Save result
write.csv(data.frame(x = x, y = y), "independent_data.csv")

message("Independent script completed successfully")

File: test_scripts/python/test1/independent.py
"""Independent script with no dependencies."""

import random

# Generate random data
data = [random.randint(1, 100) for _ in range(10)]

# Calculate correlation (fake example)
correlation = sum(data) / len(data) / 100

print(f"Starting independent analysis")
print(f"Generated {len(data)} data points")
print(f"Correlation: {correlation:.3f}")
print("Independent script completed successfully")